appservice/src/main/java/nu/yona/server/AppServiceApplication.java

```diff
@@ -54,13 +55,17 @@
 
 	public static void main(String[] args)
 	{
+		Optional<String> optional = Optional.empty();
+		System.out.println(optional.get());
 		PropertyInitializer.initializePropertiesFromEnvironment();
 		SpringApplication.run(AppServiceApplication.class, args);
 	}
 
 	@PostConstruct
 	public void initialize()
 	{
+		Optional<String> optional = Optional.empty();
+		System.out.println(optional.get());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://build.dev.yona.nu:9000/coding_rules#rule_key=squid%3AS106)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "optional.isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://build.dev.yona.nu:9000/coding_rules#rule_key=squid%3AS3655)